src/native/libs/System.Native/pal_networkstatistics.c

```diff
@@ -479,7 +492,7 @@ int32_t SystemNative_GetActiveUdpListeners(IPEndPointInfo* infos, int32_t* infoC
         }
         estimatedSize = tmpEstimatedSize;
     }
-    int32_t count = (int32_t)(estimatedSize / sizeof(struct xtcpcb));
+    int32_t count = (int32_t)(estimatedSize / sizeof(struct xinpcb));
```

Comment:
This might be incorrect, but I'm unsure. Based on [D15386](https://reviews.freebsd.org/D15386) I think this might need to be `kptr_t count = (kptr_t)...`?

Reply:
I was actually thinking about moving the count inside and simply count the iterations instead of trying to calculate it from size. I'll do more testing on macOS as well